### PR TITLE
Feat: new rule monitor_gce_network_name

### LINKED ISSUE #65: [create new rule] gce_network_name / no default_network
- @BrunoReboul 
  - [x] create an issue
- @MarcFundenberger
  - [x] create topic branch named rulename 
  - [x] create `.rego` rule file
  - [x] create `.yaml` constraint file
  - [x] create `readme.md`
  - [x] Initiate PR
- @BrunoReboul 
  - [x] review PR, iterate
  - [x] create system tests, iterate
  - [x] merge to dev

### PR DESCRIPTION
**Pull request by MarcFundenberger**
fixes #65 

New rule to check VPC Networks Name.
Rule supports blacklist / whitelist mode.
Exemptions are allowed (on asset name)
Matches on exact network name.

### CODE REVIEW

File: services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    
+    network_name := asset.resource.data.name
+    trace(sprintf("network_name: %v", [network_name]))
+
+    matches_found =  [n | n = params.name[_]; network_name == n]
+    trace(sprintf("Violation found  if %v != %v", [desired_count,count(matches)]))
+    count(matches_found) != desired_count
+
+    message := sprintf("Network %v is present.", [network_name])
```
Comment (BrunoReboul) [inline services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego:47]: Network %v is a default network.

**Review by BrunoReboul — CHANGES_REQUESTED** [review]
thanks.
A little change in the output msg

**Comment by BrunoReboul** [comment]
system test created and PASSED

File: services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego (hunk 1 above)
Comment (MarcFundenberger) [reply services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego]: done

Comment (BrunoReboul) [reply services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego]: Did you push your changes, I do not see it ?

**Review by BrunoReboul — CHANGES_REQUESTED** [review]

File: services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego (hunk 1 above)
Comment (MarcFundenberger) [reply services/monitor/instances/monitor_gce_network_name/monitor_gce_network_name.rego]: yes, you're looking at an outdated version (see commit https://github.com/BrunoReboul/ram-config-template/pull/73/commits/8e01c2b71e198806087b0778f3a4f5c26f1949db)

**Review by BrunoReboul — APPROVED** [review]

**Pull request merged**